datastructure/Administrator.py

```diff
@@ -97,7 +97,7 @@ def get_name(self) -> str:
     def encrypt_password(plain_text_password: str) -> str:
         # Hash a password for the first time
         #   (Using bcrypt, the salt is saved into the hash itself)
-        return bcrypt.hashpw(plain_text_password, bcrypt.gensalt())
+        return bcrypt.hashpw(plain_text_password.encode('utf-8'), bcrypt.gensalt())
```

Comment:
Hi @sandip-narwade, thank you for the pull request.
I've test your solution and i see no issue, only one questions before accept the pull request, cause that error is not raised running the script in my current environment: 
- Are you sure that you have installed `py-bcrypt==0.4` and not `bcrypt`?